### PR TITLE
`treehouses service pylon`

### PR DESCRIPTION
**Pull request by Neethu-Mohan**

### CODE REVIEW

**Comment by darthnoward** [comment]
image
when I first try to access pylon, I was prompted with entering env variable in /srv/pylon/.env (root, root). Is there a way to avoid this?

**Comment by Neethu-Mohan** [comment]
1. Please use 
'git pull origin master' before testing
2. the default username:root
   the default password:root

File: services/install-pylon.sh [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    echo "pylon_autorun=true"
+    echo
+    echo "if [ \"\$pylon_autorun\" = true ]; then"
+    echo "  treehouses services pylon up"
+    echo "fi"
+    echo
+    echo
+  } > /srv/pylon/autorun
+}
+
```
Comment (kjong) [inline services/install-pylon.sh:41]: Can you change the `echo`s to heredocs?

File: services/install-pylon.sh [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+# add port(s)
+function get_ports {
+  echo "3131"
+}
+
+ #add size (in MB)
+function get_size {
+ echo "255"
```
Comment (kjong) [inline services/install-pylon.sh:61]: Needs an extra space

File: services/install-pylon.sh [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+# add info
+function get_info {
+  echo "https://github.com/linuxserver/docker-pylon.git"
+  echo
+  echo "\"Pylon is a web based integrated development environment built with Node.js as a backend and with a supercharged JavaScript/HTML5 frontend\""
+}
+
+# add svg icon
+function get_icon {
```
Comment (kjong) [inline services/install-pylon.sh:72]: I don't think this is the correct icon

Comment (Neethu-Mohan) [reply services/install-pylon.sh:72]: @kjong can we go with original cloud9 icon? any suggestion?

Comment (kjong) [reply services/install-pylon.sh:69]: I'm not sure if it's too important but the current icon you have is for something different from the IDE.

**Comment by dogi** [comment]
https://github.com/treehouses/cli/pull/1576

**Pull request closed without merging**